modules/veteran/app/controllers/veteran/v0/accredited_representatives_controller.rb

```diff
@@ -32,24 +31,12 @@ def index
 
       private
 
-      def organization_search?
-        @organization_search ||= search_params[:type] == 'organization'
-      end
-
       def model_klass
-        @model_klass ||= if organization_search?
-                           'Veteran::Service::Organization'.constantize
-                         else
-                           'Veteran::Service::Representative'.constantize
-                         end
+        @model_klass ||= 'Veteran::Service::Representative'.constantize
```

Comment:
Some of the "klass" stuff isn't needed anymore now that organizations aren't searchable